cmd/virt-freezer/main.go

```diff
@@ -76,6 +76,18 @@ func main() {
 		os.Exit(1)
 	}
 
+	info, err := client.GetGuestInfo()
+	if err != nil {
+		log.Log.Reason(err).Error("Failed to get guest info")
+		os.Exit(1)
+	}
+
+	log.Log.Infof("Guest agent version is %s", info.GAVersion)
```

Comment:
The log line may be better placed at line 90.

Reply:
Yup